### ISSUE TITLE
Install latest does not work for versions > x.9.x

### ISSUE THREAD

**Issue by MaxTru** [barmac/asdf-camunda-modeler#8]
**Describe the bug**

Install latest does not work for versions > x.9.x

**Steps to reproduce**

* Be on camunda-modeler 5.9.0
* Try to install camunda-modeler 5.12.0

```
maxtru@MacBook-Pro-von-Maximilian ~ % asdf install camunda-modeler latest
camunda-modeler 5.9.0 is already installed.  <------------- Bug here
maxtru@MacBook-Pro-von-Maximilian ~ % asdf install camunda-modeler 5.12.0
* Downloading modeler 5.12.0...
######################################################################### 100,0%
modeler 5.12.0 installation was successful!
```
**Expected behavior**

Install latest works for versions > x.9.x

**Screenshots**

n/a

**Additional context**

n/a

**Comment by barmac** [comment]
Thanks for reporting! I can reproduce the problem.

**Comment by barmac** [comment]
Released as v2.0.3. Please run `asdf plugin update camunda-modeler` to verify it works on Linux. Tests are still to be written ;)

**Comment by MaxTru** [comment]
> Released as v2.0.3. Please run `asdf plugin update camunda-modeler` to verify it works on Linux. Tests are still to be written ;)

I switched to MacOS :-) It works. THanks for the super fast support! <3 asdf camunda-modeler
